packages/api-server/src/handlers/stream.ts

```diff
@@ -80,12 +80,14 @@ export function createStreamHandlers(router: SequentialCeroRouter) {
     const downstream = (
         path: string | RegExp,
         stream: StreamOutput,
-        { json = false, text = false, end: _end = false, encoding = "utf-8" }: StreamConfig = {}
+        { json: _json = false, text: _text = false, end: _end = false, encoding = "utf-8" }: StreamConfig = {}
     ): void => {
         router.post(path, async (req, res, next) => {
             try {
 
-                checkAccepts(req.headers["content-type"], text, json);
```

Comment:
We are doing checks on this, and this is a separate module that can be used to stream stuff directly. I'd add a switch that allows disabling this.

Reply:
fixed